doc/projectionintro.html

```diff
@@ -37,17 +37,17 @@ <h3>Vector map projections</h3>
 
 Vector maps are backward projected in GRASS. This means that the
 user has to run the vector projection command (<a href=v.proj.html>v.proj</a>)
-in the target location and "pull" the map from the source location.
-Both locations need to have a projection defined.
+in the target project and "pull" the map from the source project.
+Both projects need to have a projection defined.
```

Comment:
```suggestion
Vectors are reprojected using the vector projection command (<a href=v.proj.html>v.proj</a>). The command is used in the target project to "pull" a map from its source project. Both projects need to have a projection defined.
```

(same attempt at clarifying as above. Feel free to take it or leave it.)